frontend/src/App.css

```diff
@@ -12,17 +12,16 @@
 
 @media only screen and (max-width: 768px) {
   .all-Pages-Common {
-    padding-top: 17%;
+    padding-top: 15%;
   }
 }
 
 @media only screen and (max-width: 480px) {
   .all-Pages-Common {
-    padding-top: 36%;
+    padding-top: 17%;
   }
 }
```

Comment:
![image](https://user-images.githubusercontent.com/44992128/76725354-efc84180-6773-11ea-892b-b261e9936c4a.png)
I see a regression around this class, there is space between header and page content. @itsnamangoyal  please check that.